bindings/Sofa/src/SofaPython3/Sofa/Helper/CMakeLists.txt

```diff
@@ -24,5 +24,5 @@ SP3_add_python_module(
     DESTINATION  Sofa
     SOURCES      ${SOURCE_FILES}
     HEADERS      ${HEADER_FILES}
-    DEPENDS      Sofa.Core Sofa.Helper SofaPython3::Plugin
+    DEPENDS      Sofa.Core SofaPython3::Plugin
```

Comment:
Should be propagated to [Bindings.SofaConfig.cmake.in](https://github.com/sofa-framework/SofaPython3/blob/master/bindings/Sofa/Bindings.SofaConfig.cmake.in)